rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/RawTransactionResult.scala

```diff
@@ -60,10 +60,10 @@ case class GetRawTransactionResult(
     locktime: UInt32,
     vin: Vector[GetRawTransactionVin],
     vout: Vector[RpcTransactionOutput],
-    blockhash: DoubleSha256Digest,
+    blockhash: Option[DoubleSha256Digest],
```

Comment:
We had some errors in our RPC definitions. 
Bitcoin Core developer reference: https://bitcoin.org/en/developer-reference#getrawtransaction